src/legacy/core_plugins/tile_map/public/coordinate_maps_visualization.js

```diff
@@ -172,7 +168,7 @@ export function CoordinateMapsVisualizationProvider(Notifier, Private) {
         tooltipFormatter: this._geoJsonFeatureCollectionAndMeta ? boundTooltipFormatter : null,
         mapType: newParams.mapType,
         isFilteredByCollar: this._isFilteredByCollar(),
-        fetchBounds: this.getGeohashBounds.bind(this),
+        fetchBounds: () => this.vis.API.getGeohashBounds(), // TODO: Remove this (elastic/kibana#30593)
```

Comment:
I don't believe it is necessary to pass `this.vis` to the function here, as it is already available in `embedded_visualize_handler` and should be the same object, but let me know if for some reason this is an incorrect assumption.